client/app/scripts/superdesk/ui/views/weekday-picker.html

```diff
@@ -0,0 +1,6 @@
+<button class="btn btn-light pull-left"
+    ng-repeat="day in weekdayList"
+    ng-click="toggleDay(day)"
+    ng-class="{active: isDayChecked(day)}">
+    {{ weekdays[day] }}
```

Comment:
Does the name of the day already get translated when defining the `weekdays` factory and calling `gettext()`?